CONTRIBUTING.md

```diff
@@ -5,3 +5,8 @@
 - Pull requests must be accompanied by passing all existing automated tests (`$ npm test`).
 
 Thanks in advance for helping out!
+
+I have tried to add some more things into your repo like -
+
+1. Marquee tag in html5
+2. Style the page with the help of CSS like styling the image, headings and background-color etc.
```

Comment:
This message does not belong in the Contributing document, leave this to the PR message.